src/config/default.properties

```diff
@@ -5,7 +5,7 @@ db_username=vimc
 db_password=changeme
 db_driver=org.postgresql.Driver
 
-testdb_image=docker.montagu.dide.ic.ac.uk:5000/montagu-db:master
+testdb_image=docker.montagu.dide.ic.ac.uk:5000/montagu-db
```

Comment:
tiny thing - why not put the colon on the end of here, rather than appending in the gradle task, makes code in build.gradle slightly simpler and might also make it clearer that this is an incomplete image path?